src/components/ReportActionItem/ChronosOOOListActions.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
+import styles from '../../styles/styles';
+import Text from '../Text';
+import Button from '../Button';
+import * as Chronos from '../../libs/actions/Chronos';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import DateUtils from '../../libs/DateUtils';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+
+const propTypes = {
+    /** The ID of the report */
+    reportID: PropTypes.string.isRequired,
+
+    /** All the data of the action */
+    action: PropTypes.shape(reportActionPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const ChronosOOOListActions = (props) => {
+    const events = lodashGet(props.action, 'originalMessage.events', []);
+
+    if (!events.length) {
+        return (
+            <View style={[styles.flexRow, styles.alignItemsCenter, styles.pt, styles.ml18]}>
+                <Text>You haven&apos;t created any events</Text>
+            </View>
+        );
+    }
+
+    return (
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props.action, 'pendingAction', null)}
```

Comment:
Any reason `errors` are not passed here?

Reply:
I think mostly because there won't ever be any errors from the web API.